inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp

```diff
@@ -142,3 +142,48 @@ ngraph::pass::HSwishFusionWithHSigmoid::HSwishFusionWithHSigmoid() {
     auto m = std::make_shared<ngraph::pattern::Matcher>(mul_pattern, matcher_name);
     register_matcher(m, callback);
 }
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::HSwishFusionWithClampMul, "HSwishFusionWithClampMul", 0);
+
+ngraph::pass::HSwishFusionWithClampMul::HSwishFusionWithClampMul() {
+    MATCHER_SCOPE(HSwishFusionWithClampMul);
+    // Replaces a sub-graph (Clamp(x + 3, 0, 6) * x) with a HSwish * 6.
+    auto input = ngraph::pattern::any_input();
+    auto add_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
```

Comment:
Looks like some variables can be const because you don't change values.

Reply:
Yes you're right. Previously, I just follow other Hswish transformations and don't add `const`. I will add `const` to constant variables